projects/1.21/assets/merequester/merequester/lang/zh_cn.json

```
@@ -0,0 +1,32 @@
+{
+  "itemGroup.merequester.tab": "ME请求器",
+
+  "item.merequester.requester_terminal": "ME请求终端",
+  "block.merequester.requester": "ME请求器",
+
+  "gui.merequester.requester_terminal": "ME请求终端",
+  "gui.merequester.no_requesters": "没有找到ME请求器！",
+
+  "tooltip.merequester.requester_desc": "在需要时，通过请求合成，自动保持物品和流体的库存。必须连接到ME网络，可从ME请求终端进行访问。断开后将失去其配置。可以使用内存卡复制设置。",
+
+  "tooltip.merequester.shift_for_more": "按下%s获取更多信息",
+  "tooltip.merequester.enter_to_submit": "按下%s来提交更改",
+
+  "tooltip.merequester.whole_number": "必须是整数！",
+  "tooltip.merequester.locked": "已锁定！请求正在进行！",
+
+  "tooltip.merequester.toggle": "请求开关",
+  "tooltip.merequester.amount": "库存数量",
+  "tooltip.merequester.batch": "单批数量",
+  "tooltip.merequester.submit": "提交更改",
+  "tooltip.merequester.status": "请求状态",
+
+  "tooltip.merequester.idle": "空闲",
+  "tooltip.merequester.idle_desc": "当已经达到要求的数量，或没有该物品或流体的合成样板时，请求器处于空闲状态。",

```

```suggestion
  "tooltip.merequester.idle_desc": "当已经达到要求的数量，或没有该物品/流体的合成样板时，请求器将处于空闲状态。",
```
两个或感觉念起来不是很通顺，不过这里改了又和其他地方不是很一致了……